src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,463 @@
+
+There are many different ways to perform searching and filtering with table data. Below are a few examples:
+
+<ImageGrid
+    images={[Filter1, Filter2, Filter3, Filter4]}
+    captionsUnderImages={[
+        `A simple search field embedded in a table.`,
+        `A search field outside a table.`,
+        `Filter criteria listed as chips.`,
+        `A "saved filter" dropdown.`,
+    ]}
+/>
+
+<ImageGrid
+    images={[AdvancedTableFilter]}
+    caption={[
+        `You may combine multiple interactions (searching and filtering) and input methods (keyboard navigations, keyboard typing, mouse clicking) together for an "advanced search".`,
+    ]}
+    regularWidth
+/>
+
+## Transition Animation
+
+Often, table data look highly similar between rows and can disorient your users. You are encouraged to use animations in your applications to help users understand certain transitions.
+
+<ImageGrid
+    images={[AdditionAnimation, SubstractionAnimation]}
+    captionsUnderImages={[
+        `Transition animation for adding a row to the table. Consider using a background color to temporarily highlight the newly-added row.`,
+        `Transition animation for removing a row from the table.`,
+    ]}
+/>
+
+<Divider />
+
+# Tables vs. Lists
+
+<ImageGrid
+    images={[ListVTableTable, ListVTableList]}
+    caption={['A same set of data may be presented in tables (#1) or lists (#2), depends on the use cases.']}
+/>
+
+In most cases, we encourage you to organize your data into lists instead of tables. However, there is no clearly defined boundary between lists and tables. Below are a few factors you should consider when deciding how you want to present your data.
+
+## Arrangement
+
+Tables expose each data point into their own columns, allowing users to easily look up a row starting from any column. Lists, on the other hand, group their content row-by-row.
+
+<ImageGrid
+    images={[ArrangeTable, ArrangeList]}
+    captionsUnderImages={[
+        'Tables take up more space and is often the main focus of the page they are on.',
+        'Lists are more horizontally compact and are frequently used for dashboards. Typography changes can give the text more visual hierarchy.',
+    ]}
+/>
+
+Because of this, tables tend to take up a lot of horizontal space, therefore they are almost always the only main content on the page of a desktop application. You should not use tables in dashboards, where many widgets are competing for the space; you should also avoid using tables on mobile applications, where the viewport size is limited.
+
+<ImageGrid
+    images={[DashboardTable, PageBodyTable]}
+    captionsUnderImages={[
+        'AVOID: Avoid using tables in dashboards. Use lists instead.',
+        'DO: Tables are usually the only main content on a page.',
+    ]}
+/>
+
+A "tabular list" or a structured list that attempts to combine tables and lists together might result in a design that does not pick up the advantages of either pattern.
```

Comment:
I might just leave this bit out entirely if it's not something we encourage

Reply:
I've seen several teams doing it; would be good to give our official opinion on this.